### PR TITLE
(feat) UHM-8240 Create O2 Visit Summary workspace for Ward App

### PR DESCRIPTION
**Pull request by chibongho**
## Summary
Adds a new workspace (extensions) for the ward app to open up the O2 Visit Summary page for a patient. The workspace is created under a new app called `esm-commons-app`.

Tested this locally by doing `yarn start  --sources packages/esm-commons-app/  --port 8081 --backend http://localhost:8080 --config-file ../sl_local_config.json`. We'll also need the `spa-build-config.json` file in `openmrs-frontend-pihemr` to include the new app. 

I also upgraded `openmrs-esm-core`, and a few things (like `turbo`) to match versions used by `openmrs-esm-patient-management`. I had to change a few things in `esm-referrals-app` as a result. They seem benign, but I haven't tested them. (Let me know what to test for.)

## Screenshots
image
Note that there are both the "regular" O3 Visit Summary and the O2 Visit Summary in the action menu of the workspace. I'll make a corresponding config change to hide / reorder the appropriate icons.

## Related Issue
https://pihemr.atlassian.net/browse/UHM-8240

### CODE REVIEW

File: packages/esm-commons-app/translations/en.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (chibongho) [inline packages/esm-commons-app/translations/en.json:1]: Not sure why, but I had to manually add these translations in, even though there is an `extract-translations` pre-commit hook. Projects like `openmrs-esm-patient-management` automatically updates this file on commit.

**Comment by mogoodrich** [comment]
@chibongho the referrals queue can be tested on HUM-CI (I'm fine if you want to merge and test after merging). It's the "J9 Referrals" app availalbe at the Sante Famn location:

image

If you set the date back a few months, some referrals should show up:

image

File: packages/esm-commons-app/src/ward-app/o2-visit-summary-workspace.component.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    const dashboard = iframeRef.current.contentDocument;
+    const elementsToHide = ['header', '.patient-header', '#breadcrumbs', '.visit-actions', '#choose-another-visit'];
+
+    const styleTag = dashboard.createElement('style');
+    styleTag.innerHTML = elementsToHide.map((e) => `${e} {display: none;}`).join('\n');
+    dashboard.head.appendChild(styleTag);
+  }, []);
+
+  if (patient && visit) {
+    const src = `/openmrs/pihcore/visit/visit.page?patient=${patient.uuid}&visit=${visit.uuid}`;
```
[the remark this answers is not in the document]
Comment (chibongho) [reply packages/esm-commons-app/src/ward-app/o2-visit-summary-workspace.component.tsx:20]: I can't a variable for it. I'll leave it as is for now.

File: packages/esm-commons-app/translations/en.json (hunk 1 above)
Comment (mseaton) [reply packages/esm-commons-app/translations/en.json:1]: Would be good to see if we can get an answer to why that is.

File: packages/esm-commons-app/src/ward-app/o2-visit-summary-workspace.component.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (chibongho) [reply packages/esm-commons-app/src/ward-app/o2-visit-summary-workspace.component.tsx:20]: Found `openmrsBase`. I just pushed a change to make it use that.

**Pull request merged**